testing/pretty.ts

```diff
@@ -0,0 +1,75 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+
+import { equal } from "./mod.ts";
+import { red, green, white, gray, bold } from "../colors/mod.ts";
+import diff, { DiffType, DiffResult } from "./diff.ts";
+import { format } from "./format.ts";
+
+const CAN_NOT_DISPLAY = "[Cannot display]";
+
+function createStr(v: unknown): string {
+  try {
+    return format(v);
+  } catch (e) {
+    return red(CAN_NOT_DISPLAY);
+  }
+}
+
+function createColor(diffType: DiffType) {
+  switch (diffType) {
+    case "added":
+      return (s: string) => green(bold(s));
+    case "removed":
+      return (s: string) => red(bold(s));
+    default:
+      return white;
```

Comment:
Should this be `(s: string) => s` since people with light terminal themes won’t be able to read white text?